src/bolt/utils.ts

```diff
@@ -176,7 +226,78 @@ export async function sendCodeReviewSummary (channel: string): Promise<void> {
     });
 }
 
-export async function sentHomePageCodeReviewList (slackUserId: string, status: string = 'pending'): Promise<void> {
+export async function sentHomePageCodeReviewList ({slackUserId, codeReviewStatus, slackChannelId}: {slackUserId: string; codeReviewStatus?: string; slackChannelId?: string}): Promise<void> {
+    const buttonPendingReviews = {
+        type: 'button',
+        text: {
+            type: 'plain_text',
+            text: 'Pending Reviews',
+            emoji: true
+        },
+        value: `pending`,
+        action_id: `pending`
```

Comment:
Should this use straight quotes since they aren't dynamic?

Reply:
Good catch, yes, it's a typo.